extensions/arc/deployment/src/main/java/io/quarkus/arc/deployment/AnnotationsTransformerBuildItem.java

```diff
@@ -1,8 +1,22 @@
 package io.quarkus.arc.deployment;
 
+import org.jboss.jandex.AnnotationTarget;
+
 import io.quarkus.arc.processor.AnnotationsTransformer;
 import io.quarkus.builder.item.MultiBuildItem;
 
+/**
+ * Makes it possible to add, remove or alter annotations on various components.
```

Comment:
```suggestion
 * Make it possible to add, remove or alter annotations on various components.
```
is generally better.